docs/syntax.md

```diff
@@ -21,6 +25,12 @@ Below you will find the step syntax next to the name of the method it utilizes.
 - `<GK> [the] (deployment|hpa|horizontalpodautoscaler|service|pdb|poddisruptionbudget|sa|serviceaccount) <any-characters-except-(")> is in namespace <any-characters-except-(")>` kdt.KubeContext.ResourceInNamespace
 - `<GK> [I] scale [the] deployment <any-characters-except-(")> in namespace <any-characters-except-(")> to <digits>` kdt.KubeContext.ScaleDeployment
 - `<GK> [the] (clusterrole|clusterrolebinding) with name <any-characters-except-(")> should be found` kdt.KubeContext.ClusterRbacIsFound
+- `<GK> [I] (add|remove) ?<any-characters-except-(")> as trusted entity to iam role <any-characters-except-(")>` kdt.AwsContext.IamRoleTrust
```

Comment:
What is that `?` character for? i think @Kyle-Wong introduced it.

Reply:
A question mark ( ? ) immediately following a character means match zero or one instance of this character. So the space before the name is optional I think? Not a regex pro by any means so could be wrong.

Reply:
That is a mistake, that parameter is not optional. Good catch